### DISCUSSION TITLE
Automatic sync with other messenger platforms?

### DISCUSSION THREAD

**Discussion by MrCredible** [monicahq/monica#4999] [General]
I can't tell if Monica is supposed to do automatic sync of conversations and events from other platforms. Is this currently supported or planned to be supported in the future? Do you have to manually enter all interactions with other people if you want to track it in Monica?

I dont see anything specifying any type of automatic sync on github, but on the website I thought I read that the dashboard would show up all your conversations/events automatically.

**Comment by MrCredible** [comment]
@djaiss 

My apologies if its a silly question. 
Id appreciate any feedback though.

**Comment by Elliot-Construct** [comment]
At the very least, it should recognize the pasting of WhatsApp conversations into a message box and populate the individual message boxes!

> [5:20 pm, 16/08/2022] Tom: blah blah blah 
> [6:30 pm, 16/08/2022] Elliot: Blue blue blue
> [7:15 pm, 16/08/2022] Tom: Red red red red

It should then create a message box for each line!

**Comment by siraht** [comment]
I really want to commit to using Monica, but having tried before, know that I'm unable to if manual tracking of interactions is a necessity.

Is it at all a possibility for the future that Monica will integrate tracking of calls+sms on mobile devices, as well as email tracking? Whether the latter be via a BCC based email tracking method, or something else?

I'm well aware of the current limited capacity for feature development paired with the, I'm sure, consuming project of launching the Monica rewrite. And, at the very least, I'd like to plant the seed for consideration of the functionality as a desired goal, to be completed at some indeterminate point in the future. Not to ask for more, more, more after the huge task of rewriting Monica, but to allude to the economic and social market opportunity for an open source and private personal relationship tracker that takes the work out of tracking relationships, and is usable for everyone.

Thanks for all your work @djaiss :heart:
